deploy_staging.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+docker build -t gcr.io/${PROJECT_NAME_STG}/${DOCKER_IMAGE_NAME}:$TRAVIS_COMMIT ./docker
+
+echo $GCLOUD_SERVICE | base64 --decode -i > ${HOME}/gcloud-service-key.json
+cat ${HOME}/gcloud-service-key.json
+gcloud auth activate-service-account --key-file ${HOME}/gcloud-service-key.json
+
+gcloud --quiet config set project $PROJECT_NAME_STG
+gcloud --quiet config set container/cluster $CLUSTER_NAME_STG
+gcloud --quiet config set compute/zone ${CLOUDSDK_COMPUTE_ZONE}
+gcloud --quiet container clusters get-credentials $CLUSTER_NAME_STG
+
+gcloud docker -- push gcr.io/${PROJECT_NAME_STG}/${DOCKER_IMAGE_NAME}
```

Comment:
@nikhilrayaprolu you can host the docker images there and use them in your kubernetes configuration. A replacement for `gcr.io`. 

We have been doing that for quite sometime with eventyay. https://hub.docker.com/u/eventyay/